cmd/gen-crl/main.go

```diff
@@ -0,0 +1,128 @@
+package main
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/x509"
+	"encoding/pem"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path"
+	"time"
+
+	"github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/pkcs11helpers"
+)
+
+type crlDef struct {
```

Comment:
Let's use a longer type name like crlDefinition. And add a godoc comment explaining how this is used.